.travis/build-check.sh

```diff
@@ -47,8 +39,28 @@ LIBS="${SSL_LIBS} -llz4"
 #    ${CXX} ${CXXFLAGS} ${INCLUDEDIRS} ${LDFLAGS} cli.cpp -o cli ${LIBS}
 #)
 
-(
-    cd test/ssl
-    ${CXX} ${CXXFLAGS} ${INCLUDEDIRS} ${LDFLAGS} proto.cpp -o proto ${LIBS}
-    ./proto
-)
+BUILD_COMMAND="${CXX} ${CXXFLAGS} ${INCLUDEDIRS} ${LDFLAGS} proto.cpp -o proto ${LIBS}"
+
+if [ "${RUN_COVERITY_SCAN}" = "0" -o "${TRAVIS_BRANCH}" != "${COVERITY_BRANCH}" ]; then
```

Comment:
This will not run the standard tests if coverity runs. I don't think that's what you intended, because you didn't add a coverity-specific matrix entry (but instead added RUN_COVERITY=1 to one of the existing ones). So I think this check should be the other way around and not have an "else".

Reply:
Actually running coverity instead of the standard tests is what I intended. But now that I think about it again, I see it is not really correct. We should always run the standard tests and then build for coverty.
I will fix this.

Thanks!